Backend/dev/migrations/rider-app/1103-reuse-old-estimates-in-repetition.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE atlas_app.estimate ADD COLUMN bpp_estimate_id character(36);
+UPDATE atlas_app.estimate AS T1 SET bpp_estimate_id = 'UNKNOWN';
+ALTER TABLE atlas_app.estimate ALTER COLUMN bpp_estimate_id SET NOT NULL;
+
+UPDATE atlas_app.person_flow_status SET flow_status = '{"tag":"IDLE"}';
```

Comment:
> Why this? It will reset each customer to the `IDLE` state which will not be the case in prod?

Coz new select flow based on `bpp_estimate_id`, which is `'UNKNOWN'` for all old estimates, every user who is currently at GOT_ESTIMATE state will end up with error.